### ISSUE TITLE
Support Expo

### ISSUE THREAD

**Issue by slorber** [vincentriemer/react-native-dom#43]
Hi,

This would be awesome if the Expo hello world app would be easy to deploy as a website, making the dom RN platform more accessible without having to eject, install plugin, setup configuration etc...

Also many of us already have Expo non-ejected apps and it would be nice to be able to test RN-dom on it without ejecting.

I guess this also needs support from Expo team to work and it's not only on your side, just opening the issue to see if there is interest for this and track eventual progress.

See also [Canny request](https://expo.canny.io/feature-requests/p/support-react-native-dom)

**Comment by kay-is** [comment]
Sounds like a good idea, since Expo has a (rather) fixed amount of native modules.

**Comment by slorber** [comment]
@kay-is there is already some web support for Expo apis done by @raarts: https://github.com/expo/expo-sdk/pull/107

**Comment by vincentriemer** [comment]
I agree that the ability to easily run Expo apps is very important when it comes to adoption of RND but supporting Expo directly is out of the scope of this project. Expo is a layer above the react native core so it'll either be up to their team or a community-led project to add the support. For a starting point, I did vastly non-exaustive shim of expo for the react europe app I presented [here](https://github.com/vincentriemer/react-europe-2018-mobile/tree/react-europe-dom/src/expo).

I'm going to close this issue specifically but if there are any future issues with the core itself that blocks the ability to implement the Expo SDK feel free to open additional issues 👍
